css/css-fonts/font-size-adjust-013.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>CSS Test: font-size-adjust property</title>
+    <link rel="match" href="font-size-adjust-013-ref.html">
+    <link rel="help" href="https://www.w3.org/TR/css-fonts-5/#font-size-adjust-prop">
+    <style>
+        @font-face {
+            font-family: 'ahem-ex-500';
+            src: url('./resources/ahem-ex-500.otf') format('opentype');
+        }
+        @font-face {
+            font-family: 'ahem-ex-250';
+            src: url('./resources/ahem-ex-250.otf') format('opentype');
+        }
+        .wrapper {
+            --primary-font: 'ahem-ex-500';
+            --secondary-font: 'ahem-ex-250';
+        }
+        .adjusted {
+            font-size-adjust: from-font;
+        }
+        .test {
+            font-family: var(--primary-font), var(--secondary-font);
+            font-size: 50px;
+            color: peru;
+            height: 50px;
+        }
+        .primary-font {
+            font-family: var(--primary-font);
+        }
+        .secondary-font {
+            font-family: var(--secondary-font);
+        }
+        .tall-inline-block {
+            display: inline-block;
+            height: 100px;
+        }
+        .description {
+            font-family: 'Times New Roman';
+            font-size: 14px;
+        }
+        .main {
+            font-size: 16px;
+            margin-bottom: 8px;
+        }
+    </style>
+</head>
+<body>
+    <div class="description main">For this test, we are using two fonts. AhemEx500 has an aspect value (xHeight/size) of 0.5, and AhemEx250 has an aspect value of 0.25. On both fonts, all glyphs used in the test look like a square, but AhemEx500 doesn't contain 'A' (U+0041). By setting font-size-adjust from-font instead of a number value, we test if the browser automatically decides the metric value based on the primary font.</div>
+    <div class="wrapper">
+        <div class="description">1. Same glyph x in different spans with different fonts. The second one is adjusted with font-size-adjust. They should have the same size for test to pass.</div>
```

Comment:
"Same glyph x in different spans with different fonts."

...then describe more of the expected behavior here, also by describing how the two fonts differ here.

Reply:
Done.